checks.d/system_swap.py

```diff
@@ -0,0 +1,17 @@
+from checks import AgentCheck
+try:
+    import psutil
+except ImportError:
```

Comment:
I know i told you something different yesterday but after all i think we shouldn't manage the case where psutil is not present.

The only case where it's not present is when someone installs the agent from source on a system that doesn't have a compiler. And if that's the case, the check initialization will fail and report the missing import which is an acceptable behavior.